benches/bench.rs

```diff
@@ -107,6 +107,82 @@ fn iterate_mut_100k(b: &mut Bencher) {
     })
 }
 
+fn spawn_100_by_50(world: &mut World) {
+    for i in 0..2 {
+        world.spawn((Position(-(i as f32)), Velocity(i as f32), [(); 0]));
```

Comment:
Depending on the MSRV, this could be significantly simplified using const generics.

Reply:
Great idea! I don't think MSRV is a concern for benchmark code at all, so go nuts.